### DISCUSSION TITLE
Increase limit character for the SKU

### DISCUSSION THREAD

**Discussion by lezan** [etsy/open-api#886] [Ideas]
As states on title, I am handling SKUs with a length over 50 so I was wondering it is possibile to increase the limit from 39 (the current one, am I correct?) to 100 (just an example), so I can manage all my variations.

Thanks!

**Comment by Gareth064** [comment]
I'm not sure how well "Ideas" are monitored here anymore by Etsy Devs, I've been told this has now been converted to a community driven place. You'd be better off asking them via developer@etsy.com and if you get and answer then you could update this.
